### ISSUE TITLE
LCC Project Category and LCC Deliverable Types not showing up in ScienceBase

### ISSUE THREAD

**Issue by FWSJill** [adiwg/mdEditor#158]
### Before submitting an issue to I have first:

- [ ] read the [contribution guidelines](https://github.com/adiwg/mdEditor/blob/develop/CONTRIBUTING.md)
- [ ] searched for similar already [existing issue](https://github.com/adiwg/mdEditor/issues)
- [ ] read the documentation and [wiki](https://github.com/adiwg/mdEditor/wiki)

If you have performed all the above, continue describing the issue with template below.

Please:

- *be as explicit has you can*
- *if reporting a bug*
    - *describe the problem and its symptoms*
    - *explain how to reproduce*
- *attach whatever information that can help understanding the context (screen capture, log files)*
- *do your best to use correct English (re-read yourself)*

## Issue
**mdEditor Version** (find version under settings): 0.0.0+93f7fc1c

**mdEditor Web Address** (enter the URL): https://jlblcc.github.io/mdEditor/#/dashboard

### Issue Description: 
In the "Keywords" section for all of my project records (n=12) I have selected "LCC Project Category" and "LCC Deliverable Types" but these are not part of the entry when I publish in ScienceBase. 

Here is an example of a ScienceBase entry that I think should have these keywords, but doesn't. https://www.sciencebase.gov/catalog/item/59fa33cee4b0531197aff6f9

@jllong-usgs

*Provide a general description of the problem. If issue is a feature request, tell us why this feature is useful.*

### Steps to reproduce (Bug reports only):

  1. _____
  2. _____
  3. _____

### Observed Results (Bug reports only):

  *What happened?  This could be a description, log output, etc.*

### Expected Results (Bug reports only):

  *What did you expect to happen?*

**Comment by FWSJill** [comment]
Hi, this is still an issue for me. Based on looking at the scanner I think this may also be an issue for GCPO: http://calcommons.info/sb2/scansb.php?sbid=508826c4e4b0a0cec3e5a586&cmd=summary. 

This may be because we (LCC Network and GCPO) imported files using a template spreadsheet.

**Comment by jlblcc** [comment]
Yep. Unfortunately, this was an bug in the script used to generate the mdJSON from the spreadsheet. The `type` is set to "theme":

```json
    {
      "type": "theme",
      "scheme": "https://lccnetwork.org/",
      "name": "Consevation design"
    }
```

but should be

```json
    {
      "type": "LCC Project Category",
      "scheme": "https://www.sciencebase.gov/vocab/vocabulary/52dee7c5e4b0dee2a6cd6b18",
      "name": "Conservation design"
    }
```
Note that "Conservation" is also misspelled and having a different scheme is not necessarily a problem.

See https://github.com/jlblcc/sheet-to-mdjson/blob/f6e9ed700a3367d5c417a7248ed60d3920f3539f/helper.js#L281

Maybe @mmheller can help with the https://github.com/mmheller/mdEditorFileInjector

**Comment by hmaier-fws** [comment]
Old issue not related to the mdEditor. Problem was due to external script,
